server/scripts/test_reveal.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
```

Comment:
I seem to recall you wanted to check that parameter $1 was correctly provided here.

Reply:
done